extensions/rich_text_components/Svgdiagram/protractor.js

```diff
@@ -77,11 +77,17 @@ var customizeComponent = async function(modal, shape) {
   await waitFor.visibilityOf(
     modal.element(by.css('.protractor-test-saved-diagram-container')),
     'Diagram container not visible');
+  var altTextInputElement = (
+    element(by.css('[placeholder = "Description of the diagram"]')));
```

Comment:
This doesn't seem like a very robust selection method. Could we use a `protractor-test-*` tag instead? Also, can we move this selector to the top of the page with the others?

Reply:
The unicode editor is a common template used across RTE elements. The reason for not using a class selector is beacause an RTE may have multiple unicode input editors and can cause confusion. The unique way to determine what field the unicode editor is for is using the placeholder value, which is what I have used here.